csv2bufr/__init__.py

```diff
@@ -44,6 +50,20 @@
 
 LOGGER = logging.getLogger(__name__)
 
+THISDIR = os.path.dirname(os.path.realpath(__file__))
+MAPPINGS = f"{THISDIR}{os.sep}resources{os.sep}mappings"
+
+_LATEST_=36
```

Comment:
Is this the BUFR tables version?  Suggest `BUFR_VERSION`

Reply:
I've used BUFR_TABLE_VERSION.